editoast/editoast_models/src/app_health.rs

```diff
@@ -0,0 +1,13 @@
+use diesel::sql_query;
+use diesel_async::RunQueryDsl;
+
+use crate::DbConnection;
+use crate::EditoastModelsError;
+
+pub struct AppHealth;
+
+impl AppHealth {
```

Comment:
I find it weird to have the concept of "health" in the models. It's not really something about our business logic, nothing related to trains or related objects, it's purely technical. Although, I understand that you're trying to make disappear all traces of DB in `views`. Not entirely sure how to handle that. @leo any idea? I'm available to talk about it.

Reply:
Wrong user tag 😄

I agree that the health check shouldn't stay in models. I suggested below a `ping_database` function which would `SELECT 1`. This way, when looking at the `/health` endpoint it becomes clear that's everything it does. Wdyt?

Reply:
(We could also `PING` redis btw.)

Reply:
I wonder if we also should ping Redis as we're on it?